test/metric-end-results/TestHelper.ts

```diff
@@ -16,8 +17,9 @@ import { strcmp } from "../../src/parser/helper/Helper";
  * results.fileMetrics.get(formatPrintPath(inputPath, config))
  * </code></pre>
  */
-export function getParserConfiguration(
+export function getParserTestConfiguration(
```

Comment:
There is a way to replace single attributes of a default configuration object like this:
var config = {...defaultConfig, ...customConfig}; 

Attributes with the same name are overwritten by the ones of the customConfig. By that you can pass something like this:
```
const customConfig = { enforceBackwardSlashes:: false}
getParserTestConfiguration(customConfig: Partial<Configuration>): Configuration {
const defaultConfig = // .. implementation;

return {...defaultConfig, ...customConfig};
```

Reply:
Interesting. I will look into that.

Reply:
I am not sure if this works as expected with classes, especially with ones that have a non-standard constructor like `Configuration`. But I now have used this Partial<...>` utility type to merge multiple objects of the new `ConfigurationParams` interface mentioned above to achive the same goal.